editor/src/components/editor/actions/actions.ts

```diff
@@ -1064,7 +1064,7 @@ function deleteElements(targets: TemplatePath[], editor: EditorModel): EditorMod
 
       return element.children.every((childPath) => {
         const child = MetadataUtils.getElementByInstancePathMaybe(metadata.elements, childPath)
-        child == null || isElementToBeDeleted(child) || isEmptyOrContainsDeleted(child)
+        return child == null || isElementToBeDeleted(child) || isEmptyOrContainsDeleted(child)
```

Comment:
oh my god

Reply:
Ha! I'll be adding that eslint rule :grinning:

Reply:
Well this is terrifying.